libs/fluid-elements/tab-group/src/lib/tab/tab.ts

```diff
@@ -78,16 +79,15 @@ export class FluidTab extends LitElement {
    * @attr
    * @type number
    */
-  @property({ type: Number, reflect: true })
+  @property({ type: Number, reflect: false })
   get tabindex(): number {
     return this._tabindex;
   }
   set tabindex(value: number) {
     this._tabindex = value;
-    // TODO: Figure out why using the old value does not work. The attribute vanishes in the dom/is not set
     this.requestUpdate('tabindex');
   }
-  private _tabindex = 0;
+  private _tabindex = -1;
```

Comment:
Hmm, I'm not sure this is a good idea. If the tab would ever be used outside of a tabgroup, the tab is pretty much unusable. 
It would be nice if the tab would not be inaccessible by default. If the tabgroup relies on all tabindexes in the group except for the active one to be set to -1, it should be the one controlling it.

Reply:
Right